features/admin.users.v1/components/user-profile.tsx

```diff
@@ -2049,7 +2049,9 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                             : fieldName
                         )
                     }
-                    className={ resolvedRequiredValue ? "required-icon" : "" }
+                    className={ !(isUserManagedByParentOrg &&
```

Comment:
Let's use classNames package.

Ref: https://github.com/wso2/identity-apps/blob/master/docs/write-code/CODE_QUALITY.md#use-classnames-package-for-conditionally-applying-class-names-to-ui-elements